.travis.yml

```diff
@@ -2,12 +2,12 @@ services: docker
 before_install:
   # set env vars in the build settings to interact with repositories
   # see https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings
-  - docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+  - docker login -e="${DOCKER_EMAIL}" -u="${DOCKER_USERNAME}" -p="{$DOCKER_PASSWORD}"
```

Comment:
Is there a functional difference between using the brackets and not?

Also, I think there's a typo with `{$DOCKER_PASSWORD}`